R/import_redcap.R

```diff
@@ -88,33 +89,33 @@ import_redcap <- function(redcap_uri,
   # primary id will always be first in the metadata
   db_metadata$form_name[[1]] <- NA_character_
 
-  # The user may not have requested form with identifiers. We need to make sure
-  # identifiers are kept regardless. This requires adding the form with
-  # identifiers to our redcap_read_oneshot call but filtering out extra,
-  # non-identifier fields in that form
+  # The user may not have requested instrument with identifiers. We need to
+  # make sure identifiers are kept regardless. This requires adding the
+  # instrument with identifiers to our redcap_read_oneshot call but filtering
+  # out extra, non-identifier fields in that instrument
 
-  # Set default forms for API call
+  # Set default instruments for API call
```

Comment:
Hm, we're changing the comment but leaving the variable below it as `forms_for_api_call`. Should we be changing those as well? It will have a trickle out effect into other variables/functions (like `check_forms_exist` right below).

Reply:
I think the rule should be to use "form" whenever you're programming and interacting with the REDCap API, and use "instrument" whenever you're writing human readable text.

Reply:
Sounds good then, in this one case it's a little confusing in just this instance then since the comment and the variable are next to each other.